packages/utils/src/components/FeatureInterestDialog.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { Button, DialogActions, DialogContent, Link, Typography } from '@mui/material'
+import { DialogContainer, DialogTitle } from '@vscode-marquee/dialog'
+import { faDiscord } from '@fortawesome/free-brands-svg-icons/faDiscord'
+
+interface FeatureInterestDialogProps {
+  setShowCloudSyncFeature: (show: boolean) => void
+  _isInterestedInSyncFeature: (interested: boolean) => void
+}
+const FeatureInterestDialog = ({ setShowCloudSyncFeature, _isInterestedInSyncFeature }: FeatureInterestDialogProps) => {
+  const discordLink = 'https://discord.gg/BQm8zRCBUY'
```

Comment:
nitpick: move constants outside of everything that might executes multiple times to not have this piece of data to go through garbage collection.

```suggestion
const discordLink = 'https://discord.gg/BQm8zRCBUY'

interface FeatureInterestDialogProps {
  setShowCloudSyncFeature: (show: boolean) => void
  _isInterestedInSyncFeature: (interested: boolean) => void
}
const FeatureInterestDialog = ({ setShowCloudSyncFeature, _isInterestedInSyncFeature }: FeatureInterestDialogProps) => {
```